frontend/src/pages/AuthLogin/AuthLogin.tsx

```diff
@@ -82,45 +75,49 @@ export const AuthLogin: React.FC<{ showSignUp?: boolean }> = ({
       });
     }
   };
-
-  // const components = {
-  //   Footer() {
-  //     const { tokens } = useTheme();
-  //     return (
-  //       <View textAlign="center" padding={tokens.space.large}>
-  //         <Text color={tokens.colors.neutral[80]}>
-  //           &copy; All Rights Reserved
-  //         </Text>
-  //       </View>
-  //     );
-  //   },
-  // }
-
   const onClose = () => {
     setOpen(false);
   };
-
   if (process.env.REACT_APP_USE_COGNITO) {
     return (
-      <AuthForm as="div">
-        <h1>Welcome to Crossfeed</h1>
-
-        <ThemeProvider theme={amplifyTheme}>
+      <Grid container>
+        <Grid item xs={12}>
+          <Alert severity="warning">
+            <AlertTitle>
+              {' '}
+              PLATFORM NOTIFICATION: Temporary Downtime During Crossfeed
+              Migration
+            </AlertTitle>
+            <Typography variant="caption"></Typography>
+            The Crossfeed environment is moving. The migration will require a a
+            temporary downtime of approximately one week. The downtime will
```

Comment:
@cduhn17 - You are correct on this assumption. For this release, we need to remove the current message, but have this functionality available for us in the future.

@nickviola - are you able to make the change?

Reply:
The change has been made here in commit f95791bc8c6127fd366ef7d4ec06c3f0950bdcc4

Reply:
I tweaked it a bit because I got a Github warning that there was unused code so I ended up just commenting the platform notification portion out along with the now unused imports.

ad24ac6e190b00ae2449651f7053382ac954318d
e3b673516df4da1fda708b7d2176e1eb9b7c3dfc

Reply:
@ameliav - are you ready for reviews?

Reply:
@schmelz21  yes

Reply:
Here's how the screen looks now

![image](https://github.com/cisagov/crossfeed/assets/19600787/f2ef3d96-faa4-4a18-95f9-0bc730fa9df7)